OPEA-Comps/tests/test_app.py

```
@@ -0,0 +1,43 @@
+import pytest
+from fastapi.testclient import TestClient
+from app import ExampleService
+import aiohttp
```

Ruff (0.8.2)

`aiohttp` imported but unused

Remove unused import: `aiohttp`

(F401)